admin/src/components/ReactIconsSelector/index.tsx

```diff
@@ -86,25 +103,24 @@ const ReactIconsSelector: React.FC<IReactIconsSelector> = ({
 
   useEffect(() => {
     const getIconLibraries = async () => {
+      const response = await client.get("/react-icons/iconlibrary/find");
       setIconLibraries(
-        (
-          await request("/react-icons/iconlibrary/find", {
-            method: "GET",
-          })
-        ).filter((iconLibrary: IIconLibrary) => iconLibrary.isEnabled)
+        response.data.filter(
+          (iconLibrary: IIconLibrary) => iconLibrary.isEnabled
+        )
       );
     };
 
     getIconLibraries();
   }, []);
 
+
   useEffect(() => {
-    selectedIconLibrary &&
-      setSelectableIcons(
-        Object.keys(ReactIcons).filter((icon) =>
-          icon.toLowerCase().startsWith(selectedIconLibrary)
-        ) as IReactIcon[]
-      );
+    setSelectableIcons(
```

Comment:
I would really love to still have the feature to filter icon libraries.
My use case is: I usally only use one or two icon libraries in my project for consistency. But I want to be able to search through the other libraries if my prefered once do not have the icon I am looking for (or if I don't like the one there).

I like that the icons are loaded directly now, but maybe we can combine it: load all icons at the beginning, but filter them when the user selects a icon library in the dropdown!?

Reply:
@DanielPantle Completed, sorry haven't pushed yet :/